docs/pages/guides/configuration-management/configuration-management.md

```diff
@@ -74,7 +74,7 @@ Like patterns, Meshery also comes with some sample WebAssembly Filters for you t
 
 You can also import these filters manually to your provider from the [wasm-filters](https://github.com/layer5io/wasm-filters) repo.
 
-Meshery's sample application [ImageHub](guides/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
+Meshery's sample application [ImageHub](/docs/pages/guides/infrastructure-management/deploying-sample-apps.md) will let you test out configuring these filters out-of-the-box.
```

Comment:
```suggestion
Meshery's sample application [ImageHub]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
```

@prathyushmadhu please accept and use this change.

Don't include filename extension.
Don't include "/docs".
Do use `{{ site.baseurl }}`.

Reply:
@leecalcote I have made the required changes and have commited to the same PR.